src/FplBot.WebApi/Controllers/SearchController.cs

```diff
@@ -0,0 +1,40 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Fpl.Search.Searching;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace FplBot.WebApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class SearchController : ControllerBase
+    {
+        private readonly ISearchClient _searchClient;
+        private readonly ILogger<SearchController> _logger;
+
+        public SearchController(ISearchClient searchClient, ILogger<SearchController> logger)
+        {
+            _searchClient = searchClient;
+            _logger = logger;
+        }
+        
+        [HttpGet("entries/{query}")]
+        public async Task<IActionResult> GetEntries(string query)
+        {
+            try
+            {
+                var searchResult = await _searchClient.SearchForEntry(query, 10);
+                return Ok(new
+                {
+                    Hits = searchResult,
+                });
+            }
+            catch (HttpRequestException e)
```

Comment:
Appen har en global-feilhåndtering som logger exceptions, så ikke nødvendig å try-catche denne. Med mindre du spesifikt ønsker 404 istedet for 500-responser ved exceptions akkurat her da.

Reply:
copy paste 🙃